### DISCUSSION TITLE
How show list according to the tab selected in the table

### DISCUSSION THREAD

**Discussion by orebarranco** [filamentphp/filament#12247] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.62

### How can we help you?

Hello, how could I obtain the list of resources and show it according to the tab that is active in filamentphp. Ex: In the user resource, if I check the doctor tab, show the doctors table. But on the same user page?

**Comment by leandrocfe** [comment]
Sorry, could you explain more about what you need?

**Reply by orebarranco** [reply]
I need to know how to show the list of a resource in another resource. For example, I have a users resource, on the user list page I need to show the list of users. But I also need to show the list of doctors. Is this possible?
The idea I have is to use tabs on the page to list users, show 2 tabs, users and doctors. depending on the tab that is marked, show one table or another.

**Reply by orebarranco** [reply]
![image](https://github.com/filamentphp/filament/assets/106551888/ba6c5974-fba1-44da-abd3-b25e675ac170)

**Reply by leandrocfe** [reply, marked as answer]
if you have a UserResource and a DoctorResource, you could use a cluster to group the resources
https://filamentphp.com/docs/3.x/panels/clusters

**Reply by orebarranco** [reply]
No. What I need is to show the resource tables as you click on the corresponding tab.

**Reply by leandrocfe** [reply]
Do you also have a DoctorResource?

**Reply by orebarranco** [reply]
yes I have

**Reply by leandrocfe** [reply]
Take a look:

![image](https://github.com/filamentphp/filament/assets/3833889/f1ef3cc2-9e3a-4808-85f6-ebc9e4735afc)

In this example, Posts Categories e Comments are resources and I did it using clusters

https://www.youtube.com/watch?v=V909WVmyGMU

**Reply by orebarranco** [reply]
Thank you very much, this is exactly what.
